packages/dds/tree/src/test/feature-libraries/chunked-forest/chunkedForest.spec.ts

```diff
@@ -45,16 +46,15 @@ const fooKey: FieldKey = brand("foo");
 
 describe("ChunkedForest", () => {
 	testForest({
-		suiteName: "",
+		suiteName: "ChunkedForest forest suite",
```

Comment:
These names are not really combined, they are nested. I have having a group of tests with the name "" a bit confusing when looking through the test list, especially since these are not the only tests under the ChunkedForest suite.